ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -248,22 +259,32 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
 
   const isHemiX = isHemiXVariant(variant, individual)
 
-  let warning
+  let warnings = []
   if (genotype.defragged) {
-    warning = 'Defragged'
+    warnings = ['Defragged']
   } else if (!isHemiX && isHemiUPDVariant(genotype.numAlt, variant, individual)) {
-    warning = 'Potential UPD/ Hemizygosity'
+    warnings = ['Potential UPD/ Hemizygosity']
   } else if (isCompoundHet && [individual.maternalGuid, individual.paternalGuid].every(missingParentVariant(variant))) {
-    warning = 'Variant absent in parents'
+    warnings = ['Variant absent in parents']
   }
 
   if (hasCnCall) {
     const cnWarning = getGentoypeCnWarning(genotype, variant.svType, isHemiX)
     if (cnWarning) {
-      warning = warning ? `${warning}. ${cnWarning}` : cnWarning
+      warnings = warnings.concat(cnWarning)
```

Comment:
don't reassign the list. Just use `warnings.push` to update in place and make `warnings` a `const`

Reply:
Okay. Also, I found the way I used the function `.concat()` is not correct. I should use a message array for its parameter.